src/components/steps/StepInstitutionType.tsx

```diff
@@ -43,6 +45,41 @@ export default function StepInstitutionType({
     forward(selectedValue);
   };
 
+  const intisutionTypeValueFiltered = (id: string | undefined) => {
+    switch (id) {
+      case 'prod-interop':
+      case 'prod-pagopa':
+        return institutionTypeValues.filter(
+          (it) => it.labelKey === 'pa' || it.labelKey === 'gsp' || it.labelKey === 'scp'
+        );
+      case 'prod-pn':
+        return institutionTypeValues.filter((it) => it.labelKey === 'pa');
+      case 'prod-idpay':
+        return institutionTypeValues.filter((it) => it.labelKey === 'pa');
+      default:
+        return institutionTypeValues.filter(
+          (it) =>
+            it.labelKey === 'pa' ||
+            it.labelKey === 'gsp' ||
+            it.labelKey === 'scp' ||
+            it.labelKey === 'pt'
+        );
+    }
+  };
+
+  const intisutionTypeLabelFiltered = (selectedProductId: string | undefined, itValue: string) => {
```

Comment:
typo intisution al posto di institution
```suggestion
  const institutionTypeLabelFiltered = (selectedProductId: string | undefined, itValue: string) => {
```